lib/compiler/graph-compiler.js

```diff
@@ -462,7 +462,7 @@ var GraphCompiler = CodeGenerator.extend({
     gen_Sink: function(ast) {
         // silly special case for ast.name due to sink's special-case parsing
         // and AST.
-        var proc = this.gen_proc(ast, 'clientsink', '{name: "'+ast.name+'"}');
+        var proc = this.gen_proc(ast, 'clientview', '{name: "'+ast.name+'"}');
```

Comment:
Actually looking at this further, I see there are a number of places in the tests etc where we now reference `clientview` instead of `clientsink` so my preference would be to change them all to `view` once and for all rather than wait for another PR to rename it again.